docs/installation.md

```diff
@@ -4,115 +4,145 @@ This page explains how to prepare your environment for running the bot.
+
+# Enter downloaded directory
 cd freqtrade
-# git checkout stable  # Optional, see (1)
-./setup.sh --install
+
+# your choice (1)
+git checkout stable
+
+# your choice (2)
+git checkout develop
 ```
 
-(1) This command switches the cloned repository to the use of the `stable` branch. It's not needed if you wish to stay on the `develop` branch. You may later switch between branches at any time with the `git checkout stable`/`git checkout develop` commands.
+(1) This command switches the cloned repository to the use of the `stable` branch. It's not needed, if you wish to stay on the (2) `develop` branch.
 
-## Easy Installation Script (Linux/MacOS)
+You may later switch between branches at any time with the `git checkout stable`/`git checkout develop` commands.
+
+#### Notes to /setup.sh script (Linux/MacOS)
 
 If you are on Debian, Ubuntu or MacOS Freqtrade provides the script to install, update, configure and reset the codebase of your bot.
 
 ```bash
-$ ./setup.sh
+./setup.sh -i
+./setup.sh -u
+./setup.sh -r
+./setup.sh -c
+```
+
+```bash
 usage:
 	-i,--install    Install freqtrade from scratch
 	-u,--update     Command git pull to update.
 	-r,--reset      Hard reset your develop/stable branch.
 	-c,--config     Easy config generator (Will override your existing file).
 ```
 
-** --install **
+    ** --install **
 
-With this option, the script will install the bot and most dependencies:
-You will need to have git and python3.7+ installed beforehand for this to work.
+    With this option, the script will install the bot and most dependencies:
+    You will need to have git and python3.7+ installed beforehand for this to work.
 
-* Mandatory software as: `ta-lib`
-* Setup your virtualenv under `.env/`
+    * Mandatory software as: `ta-lib`
+    * Setup your virtualenv under `.env/`
 
-This option is a combination of installation tasks, `--reset` and `--config`.
+    This option is a combination of installation tasks, `--reset` and `--config`.
 
-** --update **
+    ** --update **
 
-This option will pull the last version of your current branch and update your virtualenv. Run the script with this option periodically to update your bot.
+    This option will pull the last version of your current branch and update your virtualenv. Run the script with this option periodically to update your bot.
 
-** --reset **
+    ** --reset **
 
-This option will hard reset your branch (only if you are on either `stable` or `develop`) and recreate your virtualenv.
+    This option will hard reset your branch (only if you are on either `stable` or `develop`) and recreate your virtualenv.
 
-** --config **
+    ** --config **
 
-DEPRECATED - use `freqtrade new-config -c config.json` instead.
+    DEPRECATED - use `freqtrade new-config -c config.json` instead.
 
-### Activate your virtual environment
+------
 
-Each time you open a new terminal, you must run `source .env/bin/activate`.
+## Common Installation
```

Comment:
I find "Common installation" as title for this section wrong.

This is not the common guide, but the "manual" guide (when you can't or don't want to use the install script).
By having this named "common" suggests that this will always be needed, which it really isn't.

Reply:
Manual : OK